go/arrow/cdata/cdata.go

```diff
@@ -654,6 +680,40 @@ func (imp *cimporter) importStringLike(offsetByteWidth int64) (err error) {
 	return
 }
 
+func (imp *cimporter) importBinaryViewLike() (err error) {
+	if err = imp.checkNoChildren(); err != nil {
+		return
+	}
+
+	buffers := make([]*memory.Buffer, len(imp.cbuffers)-1)
+	// XXX couldn't figure out how to extract file_reader.go::releaseBuffers as a utility
+	defer func() {
+		for _, buf := range buffers {
+			if buf != nil {
+				buf.Release()
+			}
+		}
+	}()
```

Comment:
we could export it (just make the first letter capitalized) but if we do that, it might make more sense for it to exist inside of `memory` as a utility rather than from the `ipc` package

Reply:
I'll try that